src/main/java/ch/uzh/ifi/hase/soprafs22/entity/Chat.java

```diff
@@ -46,17 +45,30 @@ public List<Message> getMessages(int fromIndex, int toIndex) {
     // now, fromIndex must be < size, and toIndex too.
     return messages.subList(fromIndex, toIndex);
   }
+    /**
+     * Gets one Message from the chat
+     * @param index: the index of the message
+     * @return Message: the Message on position index
+     */
+    public Message getMessages(int index){
+        if (messages.isEmpty()){
+            return null;
```

Comment:
Please don't return null here, but rather Collections.emptyList(). Actually, this method is unnecessary, you can just call the above method with getMessages(fromIndex, fromIndex). (Recommendation)

Reply:
ok. so we will return an empty list, if no messages are written yet. i was no sure what we want in the frontend. i like the empty list moree than null